org-roam-db.el

```diff
@@ -597,7 +597,10 @@ database, see `org-roam-db-sync' command."
       ('filenotify
        (cl-pushnew
         (cons org-roam-directory
-              (fnr-add-watch org-roam-directory '(change) #'org-roam-db-autosync--filenotify-update))
+              (fnr-add-watch org-roam-directory
+                             '(change)
+                             #'org-roam-db-autosync--filenotify-update
+                             "\\`\\.")) ; Ignore directories that start with "."
```

Comment:
This won't ignore hidden directories within subdirectories. Not sure you do this intentionally though.